### PR TITLE
Introduction of Bastion

### PR DESCRIPTION
**Pull request by oScape**
Take a look: #2

### CODE REVIEW

File: src/ch00-00-introduction.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+This book will introduce you Bastion, a highly-available, fault-tolerant runtime system with dynamic, dispatch-oriented, lightweight process model.
+
+It supplies actor-model-like concurrency with a lightweight process implementation and utilizes all of the system resources efficiently guaranteeing of at-most-once message delivery.
+
+## Discussion and Development
+
+We use [**Discord**][Discord] for development discussions. Also please don't hesitate to open issues on [**GitHub**][GitHub] ask for features, report bugs, comment on design and more! More interaction and more ideas are better!
+
+## Contributing to Bastion Open Source Helpers
```
Comment (vertexclique) [inline src/ch00-00-introduction.md]:
```suggestion
## Contributing to Bastion
```

File: src/ch00-00-introduction.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements and ideas are welcome.
+
+A detailed overview on how to contribute can be found in the [**CONTRIBUTING guide**][contribution] on GitHub.
+
+Like bastion, this book is fault-tolerant and open source. If you find something wrong or a typo, [**send us a pull request!**][book-repo]
```
Comment (vertexclique) [inline src/ch00-00-introduction.md]:
```suggestion
Like bastion, this book is open-source. If you find something wrong or a typo, [**send us a pull request!**][book-repo]
```

File: src/ch00-00-introduction.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Bastion
+
+This book will introduce you Bastion, a highly-available, fault-tolerant runtime system with dynamic, dispatch-oriented, lightweight process model.
```
Comment (cbonaudo) [inline src/ch00-00-introduction.md]: introduce you **to** Bastion

File: src/ch00-00-introduction.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+It supplies actor-model-like concurrency with a lightweight process implementation and utilizes all of the system resources efficiently guaranteeing of at-most-once message delivery.
```
Comment (cbonaudo) [inline src/ch00-00-introduction.md]: guaranteeing _ at-most-once (remove the of)

File: src/ch00-00-introduction.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+## Discussion and Development
+
+We use [**Discord**][Discord] for development discussions. Also please don't hesitate to open issues on [**GitHub**][GitHub] ask for features, report bugs, comment on design and more! More interaction and more ideas are better!
```
Comment (cbonaudo) [inline src/ch00-00-introduction.md]: **,** ask for features ( , )

**Review by cbonaudo — CHANGES_REQUESTED** [review]

File: src/ch00-00-introduction.md (hunk 3 above)
Comment (oScape) [reply src/ch00-00-introduction.md]: Both are pretty same, btw I prefer talking about a presentation rather than a discovery.

Comment (vertexclique) [reply src/ch00-00-introduction.md]: Grammatically you need to put `to`. :) That's correct like that.

Comment (oScape) [reply src/ch00-00-introduction.md]: Oh ok, I was focus on the meaning of the sentence, not grammatically. I am a bit weird on it. Thx mates :)

Comment (cbonaudo) [reply src/ch00-00-introduction.md]: I think the correct grammar for your meaning would be "Introduce Bastion to you"

Comment (vertexclique) [reply src/ch00-00-introduction.md]: yes but it doesn't matter atm, this is trivial, just apply grammar fix and carry on. not a big deal tbh.

**Review by vertexclique — APPROVED** [review]
After @cbonaudo 's comments fixed this can be merged. For deployment talk with @o0Ignition0o about how gitbooks are built and deployed. What are the conventions etc.

**Pull request merged**